seeding/seeds/seeds.js

```diff
@@ -0,0 +1,13 @@
+const { uploadSecrets } = require("./github-secrets");
+
+const seeds = [
+  {
+    key: "setGithubSecrets",
```

Comment:
I would move this whole config in the step file. the step file will export the function and the config json, making edits to this file very small.